composer.json

```diff
@@ -32,18 +32,18 @@
         }
     },
     "require": {
-        "php": "~8.0.0 || ~8.1.0 || ~8.2.0",
-        "laminas/laminas-eventmanager": "^3.5",
-        "laminas/laminas-servicemanager": "^3.15.1",
-        "laminas/laminas-stdlib": "^3.10.1"
+        "php": "~8.1.0 || ~8.2.0 || ~8.3.0",
+        "laminas/laminas-eventmanager": "^3.12",
+        "laminas/laminas-servicemanager": "^3.22.1",
+        "laminas/laminas-stdlib": "^3.18.0"
```

Comment:
Please don't bump direct dependency versions unless there is a good reason. Composer will automatically figure out the correct version to use for the current platform.

Reply:
done